guide/app_star_catalogue.tex

```diff
@@ -209,32 +204,50 @@ \subsubsection{Star Data Records}%\label{star-data-records}
 \ref{tab:StarDataRecord0}, \ref{tab:StarDataRecord1} and \ref{tab:StarDataRecord2}
 for record structure details.
 
-It should be noted that although the positional data loses accuracy as
-one progresses though the Star Record Types, this is compensated for by
-the face that the number of zones is much higher for the files where the
-smaller precision position fields are used, so the actual resolution on
-the sky isn't significantly worse for the type 1 and 2 records in
-practice.
+Details about vector algebra related to astrometric calculations can be found in \citep{2012asas.book.....V}. For catalog with Star Data Record Type 0, in 
+practice to convert \texttt{x0}, \texttt{x1}, \texttt{x2} to right accension $\alpha$ and declination $\delta$, one can use the following formulas:
+
+\begin{equation}
+\begin{aligned}
+\alpha &= \arctan2(\texttt{x1}, \texttt{x0})\\
+\delta &= \arcsin(\texttt{x2})
+\end{aligned}
+\end{equation}
+
+and to convert proper motion vector (\texttt{dx0}, \texttt{dx1}, \texttt{dx2}) to proper motion in right accension $\mu_{\alpha^*}$ and declination $\mu_{\delta}$, one can use the following formulas:
+
+\begin{equation}
+\begin{aligned}
+	xy &= \sqrt{\texttt{x0}^2 + \texttt{x1}^2}; \\
+	\vec{p} &= \left(-\frac{\texttt{x1}}{xy}, \frac{\texttt{x0}}{xy}, 0.\right); \\
+	\vec{q} &= \left(-\frac{\texttt{x0} \cdot \texttt{x2}}{xy}, -\frac{\texttt{x1} \cdot \texttt{x2}}{xy}, xy\right);\\
+	\mu_{\alpha^*} &= \vec{p} \cdot \left(\texttt{dx0}, \texttt{dx1}, \texttt{dx2}\right); \\
+	\mu_{\delta} &= \vec{q} \cdot \left(\texttt{dx0}, \texttt{dx1}, \texttt{dx2}\right);
+\end{aligned}
+\end{equation}
+
+where $\mu_{\alpha^*}\equiv\mu_{\alpha}\cos{(\delta)}$.
 
 \begin{table}[p]
 \begin{tabular}{llllp{55mm}}\toprule
```

Comment:
After these changes, the paragraph width could be enlarged to 75mm. (55mm-->75mm)